scripts/RunCombineFits.py

```diff
@@ -387,13 +393,14 @@ def RandomStringTag(size=6,chars=string.ascii_uppercase+string.ascii_lowercase+s
     ImpactCommand = "combineTool.py -M CollectGoodnessOfFit --input higgsCombine.saturated.GoodnessOfFit.mH125.root higgsCombine.saturated.toys.GoodnessOfFit.mH125.*.root -o "+GOFJsonName
     os.system(ImpactCommand)
 
-    ImpactCommand = "python ../../../CombineTools/scripts/plotGof.py --statistic saturated --mass 125.0 "+GOFJsonName+" --title-right='' --output='saturated' --title-left='e#mu'"
+    ImpactCommand = "python ../../../CombineTools/scripts/plotGof.py --statistic saturated --mass 125.0 "+GOFJsonName+" --title-right='' --output='saturated' --title-left='#mu#tau'"
     os.system(ImpactCommand)
 
     for year in args.years:
        for channel in args.channels:
           CardNum = 1
-          TheFile = ROOT.TFile(os.environ['CMSSW_BASE']+"/src/auxiliaries/shapes/smh"+year+channel+".root")
+          TheFile = ROOT.TFile(os.environ['CMSSW_BASE']+"/src/auxiliaries/shapes/smh"+year+channel+"_GOF_nocorrelation.root")
```

Comment:
Again, this needs to change, you need to run cards through the decorrelator first